spec/system/show_game_rooms_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'ShowGameRooms', type: :system do
+  include_context 'when user is logged in'
+  let(:user) { build(:user) }
```

Comment:
You can remove this since user was already created in the shared context `spec/support/sign_in_user.rb`:) 
```suggestion
```

Reply:
Thanks Anna! HYG! 
https://github.com/lea927/drop-that-beat/pull/51/commits/84315f01a06047f64e5a15954790cab4d7d49ce1